document-service-api/src/lib/config.js

```diff
@@ -5,8 +5,17 @@
  * config should be driven by environment variables where different
  * values are required
  */
+const ONE_MEGABYTE = 1024 * 1024;
 
 module.exports = {
+  azure: {
+    BLOB_STORAGE_CONNECTION_STRING: process.env.AZURE_BLOB_STORAGE_CONNECTION_STRING,
+    BLOB_STORAGE_CONTAINER_NAME: 'documentservice'
+  },
+  documentStorage: {
+    directory: process.env.DOCUMENT_STORAGE_DIR,
+    maxFileSizeInBytes: 50 * ONE_MEGABYTE
```

Comment:
This should be `maxFileSizeInBytes: Number(process.env.MAX_FILE_SIZE_IN_BYTES)` - the application should neither know nor care of the number as that's a deployment thing